scripts/marketing_data_test.py

```diff
@@ -284,12 +295,20 @@ def marketing_dml_test(
     # treat_est_df = treat_df.drop('treat_id', axis=1)
     # Effect of the treatments trained on
     x_cols = list(X.columns.values)
-    t_cols = list(treat_df.columns.values)
-    t_cols.remove('treat_id')
+    t_cols = list(T.columns.values)
```

Comment:
We can't use `treat_df` anymore because we removed collinear columns. Switch to `T` instead.